ci-operator/step-registry/baremetal/lab/pre/pull-artifacts/baremetal-lab-pre-pull-artifacts-commands.sh

```diff
@@ -38,7 +38,31 @@ function pull_artifacts() {
 EOF
 }
 
+function pull_artifacts_day2() {
```

Comment:
Can we append `"_2"` to images if `${ADDITIONAL_WORKERS_DAY2}` is true  in `function pull_artifacts`?

Reply:
yes